1-exercises/G-array-methods/exercise.js

```diff
@@ -4,14 +4,15 @@
 */
 
 let numbers = [3, 2, 1];
-let sortedNumbers; // complete this statement
+let digits = [...numbers];  // used the spread operator to preserve the original array content   
```

Comment:
Good solution. However, the solution can still work without the use of spread operator.